File: Editorial/0000-0100/0068_Text_Justification/TypeScript/1_Approach.ts
function getWords(i: number, words: string[], maxWidth: number) {
    let currentLine = [];
    let currLength = 0;
    while (i < words.length && currLength + words[i].length <= maxWidth) {
        currentLine.push(words[i]);
        currLength += words[i].length + 1;
        i++;
    }
    return currentLine;
}

File: Editorial/0000-0100/0068_Text_Justification/CPP/1_Approach.cpp
vector<string> getWords(int i, vector<string>& words, int maxWidth) {
    vector<string> currentLine;
    int currLength = 0;
    while (i < words.size() && currLength + words[i].size() <= maxWidth) {
        currentLine.push_back(words[i]);
        currLength += words[i].size() + 1;
        i++;
    }
    return currentLine;
}
